fetchit_challenge/launch/fetchit_atrezzo.launch

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<launch>
+  <arg name="world_name" default="$(find fetchit_challenge)/worlds/fetchit_challenge_atrezzo.world"/>
```

Comment:
what is `_atrezzo` ?

Reply:
Can we replace `_atrezzo` with not a new launch file but just use the existing fetchit launch file?

Also, I think that's Spanish?